cmd/out/main.go

```diff
@@ -47,6 +48,10 @@ func main() {
 		version out.Version
 	)
 
+	// Always print a newline to the log. This allows to know _when_ we started
+	// long-running operations (for example, waiting to acquire a lock).
+	fmt.Fprintf(lockPool.Output, "Starting\n")
```

Comment:
nit picking, but the capital S looks a bit out of place :)

how about something like:

```
acquiring lock on: testers
waiting for lock...
acquired!
```